shaka/src/public/ShakaPlayerView.mm

```diff
@@ -28,6 +28,7 @@ @interface ShakaPlayerView () {
   CALayer *_avPlayerLayer;
   ShakaPlayer *_player;
   shaka::VideoFillMode _gravity;
+  CGImageRef _image;
```

Comment:
I think it would be better to store the `image_bounds` instead.  This is a raw pointer that isn't ref-counted.  If the image is freed without our knowledge, this will be a dangling pointer.

Reply:
Done